NineChronicles.Headless/Controllers/GraphQLController.cs

```diff
@@ -127,6 +131,55 @@ public async Task<IActionResult> CheckPeer([FromBody] CheckPeerRequest request)
             }
         }
 
+        [HttpGet(CheckTxId)]
+        public IActionResult CheckTx([FromQuery] string txId)
+        {
+            byte[] txIdByteArray = Enumerable.Range(0, txId.Length)
+                     .Where(x => x % 2 == 0)
+                     .Select(x => Convert.ToByte(txId.Substring(x, 2), 16))
+                     .ToArray();
+            var txIdObject = new TxId(txIdByteArray);
+            if (StandaloneContext.Store is null)
+            {
+                return NotFound("Store is null.");
+            }
+            else
+            {
+                return Ok(StandaloneContext.Store.ContainsTransaction(txIdObject));
+            }
+        }
+
+        [HttpGet(CheckTxId + "/block")]
+        public IActionResult CheckTx([FromQuery] string txId, [FromQuery] string blockHash)
+        {
+            byte[] txIdByteArray = Enumerable.Range(0, txId.Length)
+                     .Where(x => x % 2 == 0)
+                     .Select(x => Convert.ToByte(txId.Substring(x, 2), 16))
+                     .ToArray();
+            var txIdObject = new TxId(txIdByteArray);
```

Comment:
```suggestion
            var txIdObject = new TxId(ByteUtil.ParseHex(txId));
```

[`ByteUtil.ParseHex`](https://docs.libplanet.io/0.11.1/api/Libplanet.ByteUtil.html#Libplanet_ByteUtil_ParseHex_System_String_) 를 사용하는 건 어떨까요?

Reply:
간단한 함수가 있었군요 반영하겠습니다 감사합니다! txIdObject 변수명도 blockHashObject에 말씀하신 것처럼 txIdHex 정도로 변경하는 것이 좋을까요?